projects/core/src/i18n/i18next/i18next-init.ts

```diff
@@ -1,18 +1,26 @@
 import { HttpClient } from '@angular/common/http';
-import i18next, { InitOptions } from 'i18next';
+import { Injectable, OnDestroy } from '@angular/core';
+import globalI18next, { i18n, InitOptions } from 'i18next';
 import i18nextXhrBackend from 'i18next-xhr-backend';
+import { Subscription } from 'rxjs';
 import { ConfigInitializerService } from '../../config/config-initializer/config-initializer.service';
 import { LanguageService } from '../../site-context/facade/language.service';
 import { TranslationResources } from '../translation-resources';
 
 export function i18nextInit(
+  i18next: i18n,
   configInit: ConfigInitializerService,
   languageService: LanguageService,
   httpClient: HttpClient,
-  serverRequestOrigin: string
+  serverRequestOrigin: string,
+  siteContextI18nextSynchronizer: SiteContextI18nextSynchronizer
 ): () => Promise<any> {
   return () =>
     configInit.getStableConfig('i18n').then((config) => {
+      // SPIKE TODO REMOVE:
+      window['globalI18next'] = globalI18next;
+      window['i18next'] = i18next;
```

Comment:
@Platonn  - we're very interested in this fix for one of our customer, so if you don't mind i'll add some comments to make sure I understand the fix.

In this particular case, wouldn't this fail in SSR due to window being undefined?

Reply:
Hi @CarlEricLavoie, no worries, those lines will be removed before merge as it's work in progress at the moment.